Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -115,6 +115,7 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
     open override func render() {
         containers.forEach(renderContainer)
         addToContainer()
+        trigger(.coreReady)
```

Comment:
just wondering, why not use the existing `.ready` event? As it is triggered in core, it is already bound to it.

Reply:
Because `.ready` is being used by AVFoundationPlayback only. We decided to use the same event that Clappr web is using, which is `coreReady`.
And I'm not seeing `.ready` being fired of by core 🤔